src/translations/en.json

```diff
@@ -273,6 +274,123 @@
     "descLfAllCountries": "All countries",
     "descLfCountriesToBalance": "Countries participating in balancing: ",
 
+    "DynaFlow_svcRegulationOn": "Static Var Compensator regulation on",
```

Comment:
The traduction keys should match with the keys in the JSON of specific-parameters. 
And you want to do the translations now ? 
It is said in the US that it will be done later

Reply:
yes I remove all of them